<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayerWidget</class>
 <widget class="QWidget" name="AudioPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <widget class="QCustomPlot" name="audioPlot" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbPlay">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdTime">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
